test/golden/routes_golden_test.dart

```diff
@@ -152,6 +152,16 @@ void main() {
         await tester.pumpAndSettle();
       }, goldenCaptureCallback: () => tester.screenMatchesGolden(tester, 'selection_route.selection_route'));
     });
+    testAppGoldens('selection_route_settings', (WidgetTester tester) async {
+      await tester.runAppTest(() async {
+        HomeRouter.instance.goto(HomeRoutes.factory.content<Playlist>(playlistWith()));
+        await tester.pumpAndSettle();
+        await tester.tap(find.byIcon(Icons.add_rounded).first);
+        await tester.pumpAndSettle();
+        await tester.tap(find.byIcon(Icons.settings_rounded).hitTestable().first);
```

Comment:
use `find.descendant` instead, so it is semantically clear what button we expect to be pressed